test/e2e_source_map_test.ts

```diff
@@ -160,6 +160,32 @@ describe('source maps', () => {
 
     expect(sourceMap.originalPositionFor(methodPosition).line).to.equal(6, 'method position');
   });
+
+  it('composes inline sources', function() {
+    const sources = new Map<string, string>();
+    sources.set('input.ts', `
+      class X { field: number; }
+      let x : string = 'a string';
+      let y : string = 'another string';
+      let z : string = x + y;`);
+
+    // Run tsickle+TSC to convert inputs to Closure JS files.
+    const {compiledJS, sourceMap} = compile(sources, undefined, undefined, true);
+
+    const {line: stringXLine, column: stringXColumn} = getLineAndColumn(compiledJS, 'a string');
```

Comment:
I think this would read better as

```
{
  const {line, column} =  ...
  expect ... sourceMap.originalPositionFor({line, column})...
}
{
  const {line, column} =  ...
  expect ... sourceMap.originalPositionFor({line, column})...
}
```